budget_manager/forms.py

```diff
@@ -0,0 +1,14 @@
+from django import forms
+from budget_manager import models
+
+
+class IncomeForm(forms.ModelForm):
+    class Meta:
+        model = models.Income
+        fields = '__all__'
```

Comment:
Wg mnie lepiej wypisać wszystkie pola, wiem, że to zadziała, ale `Explicit is better than implicit`